ood-portal-generator/share/maintenance.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
```

Comment:
`<html lang="en">`

I need to fix this in nginx_stage html examples as well.

Reply:
Just did for nginx_stage: https://github.com/OSC/ondemand/commit/1f0bfb7a60ce8baa48f70749fbf3bfa598676c7b

Reply:
Change made